src/rules/patternDetector.js

```diff
@@ -1,68 +1,192 @@
 "use strict";
 const fs = require("fs");
-const ioUtils = require("../lib/ioUtils");
-const appRoot = require("app-root-path");
-const script = appRoot.path + "/deps/py/pattern_matcher.py";
-const patternMatcherResults =
-  appRoot.path + "/deps/py/pattern-matcher-results.json";
 const ruleType = "FMNCP";
-const exec = require("child_process").exec;
-
 let logger = require("bunyan");
-let log = logger.createLogger({ name: "std-gov-oos" });
+let log = logger.createLogger({ name: "standardly" });
```

Comment:
good catch. do you mind creating an issue for moving to this logger name for a separate pr